File: go/raindrops/raindrops.go
package raindrops

import (
	"strconv"
)

const testVersion = 2

func Convert(num int) string {
	var text string

	if num%3 == 0 {
		text = text + "Pling"
	}
	if num%5 == 0 {
		text = text + "Plang"
	}
	if num%7 == 0 {
		text = text + "Plong"
	}
	if text == "" {
		text = strconv.Itoa(num)
	}
	return text
}

File: python/raindrops/raindrops.py
def convert(number):
    if number % 3 != 0 and number % 5 != 0 and number % 7 != 0:
        return str(number)

    sound = ""
    if number % 3 == 0:
        sound += "Pling"
    if number % 5 == 0:
        sound += "Plang"
    if number % 7 == 0:
        sound += "Plong"

    return sound
